docs/config.md

```diff
@@ -9,11 +9,8 @@ This is the type of documentation output to generate.
 
 > In both instances, the source markdown file will be generated in `resources/docs/source`.
 
-If you're using `laravel` type, you can call `\Mpociot\ApiDoc\ApiDoc::routes()` from your routes file (usually `routes/web.php`). This method will create a `/doc` route for your documentation, along with a `/doc.json` variant that will return the Postman collection, if you have that enabled. This method returns the route, so you can call additional methods to customise it (by adding middleware, for instance). You can also pass in the path you'd like to use instead.
+If you're using `laravel` type, the routes will be automatically loaded. By default, routes are displayed along the `/doc` path for documentanion and `/doc.json` variant for the Postman collection. You can also specify middlewares for the routes in `middleware` parameter in `config/apidoc.php` file.
```

Comment:
Isn't this only true if the service provider was registered? We aren't suing auto-discover, so you have to document that they must register the provider.

Reply:
The package is already loaded: https://github.com/mpociot/laravel-apidoc-generator/blob/master/composer.json#L59